docs/examples/use_cases/paddle/tsm/utils.py

```diff
@@ -92,9 +92,9 @@ def load_weights(exe, prog, url):
     weight_path = _download_weight(url)
 
     if os.path.isdir(weight_path):
-        fluid.io.load_vars(
+        paddle.static.io.load_vars(
             exe, weight_path, prog,
             predicate=lambda v: os.path.exists(
                 os.path.join(weight_path, v.name)))
     else:
-        fluid.io.load_params(exe, '', prog, filename=weight_path)
+        paddle.distributed.io.load_persistables(exe, '', prog, filename=weight_path)
```

Comment:
nitpick: missing new line at the end of file

Reply:
Done